.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Maybe add this file to `.gitignore` ?

Reply:
https://editorconfig.org
This is to help keep everybody consistent. There's lots of projects on the CF GitHub that have them

Reply:
Isn't it a bit redundant with cargo-fmt ?

Reply:
I actually added this because I noticed inconsistent line endings. So it handles whitespace + character set things that "cargo fmt" doesn't, and handles non-Rust files.

Reply:
@kornelski what do you think. Is this being used in rust projects?

Reply:
I've heard of it. I personally don't use it, but it seems harmless to include the file.